Dockerfile

```
@@ -23,8 +23,17 @@ RUN \
     command -v lvm && \
     command -v mdadm
 
+# Install the Amazon SSM Agent
+RUN \
+    dnf -yq install 'dnf-command(download)' cpio rsync && \
```

nit. for more consistency
```suggestion
RUN dnf install -y \
    'dnf-command(download)' \
    cpio \
    rsync && \
```